code/script.js

```diff
@@ -20,6 +105,7 @@ const showMessage = (message, sender) => {
     `
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
   } else if (sender === 'bot') {
+    audio.play();
```

Comment:
I was surprised when the sound played the first time! It was a satisfying experience interacting with the bot partially because of this sound.